### ISSUE TITLE
need to add --env for zos help container 

### ISSUE THREAD

**Issue by Dina-Abd-Elrahman** [threefoldtecharchive/zosctl#53]
- we need to add --env option to the output of ``` zos help container ```

``` 
  zos container new --name=<container> --root=<rootflist> [--hostname=<hostname>] [--ports=<ports>] [--sshkey=<sshkey>] [--privileged] [--ssh]
    creates a new container 

```

**Comment by xmonader** [comment]
fixed here fb99a5b07721a0c872efdff5f30bdc481f095023

**Comment by Dina-Abd-Elrahman** [comment]
# ZOS Version :
119f136b55ffa16a67688a933f4ec5cd2378b17e

# Verification steps:
- ./zos help container
```
 zos container new --name=<name> --root=<rootflist> [--hostname=<hostname>] [--ports=<ports>] [--env=<envvars>] [--sshkey=<sshkey>] [--privileged] [--ssh]
    creates a new container 
```
